src/blueprint/operators/grpc.rs

```diff
@@ -121,6 +121,7 @@ pub struct CompileGrpc<'a> {
 
 struct GrpcMethod {
     pub id: String,
+    pub package_and_service: String,
```

Comment:
Why not just call it package? And keep as an array of strings?

Reply:
We could call `package`, but it also holds the name of service